### ISSUE TITLE
New Install - > htmlspecialchars() expects parameter 1 to be string, array given

### ISSUE THREAD

**Issue by HRR1337** [justkidding96/statamic-aardvark-seo#123]
When installing on Statamic 3.3 I get the following error:

`htmlspecialchars() expects parameter 1 to be string, array given (View: D:\laragon\www\bibleblog\vendor\withcandour\aardvark-seo\resources\views\cp\nav\redirects.blade.php)`

On the line it says: 

` return htmlspecialchars($value ?? '', ENT_QUOTES, 'UTF-8', $doubleEncode);`

How to solve this?

**Comment by AndrewHaine** [comment]
Hi @HRR1337 ,

Please could you provide the output of `php please support:details` so that we can try to replicate this issue.

Thanks!
Andrew

**Comment by HRR1337** [comment]
> Hi @HRR1337 ,
> 
> Please could you provide the output of `php please support:details` so that we can try to replicate this issue.
> 
> Thanks! Andrew

Statamic 3.3.4 Pro
Laravel 8.83.6
PHP 7.4.19
goldnead/statamic-toc 1.1.0
statamic/ssg 1.0.0

**Comment by AndrewHaine** [comment]
Hi @HRR1337 ,

I've been unable to replicate this using the details provided, doing a little searching around though I've found that other folks have reported that this could be an error with the translation functionality in the control panel ([This issue for example](https://github.com/statamic/cms/issues/1108)). 

What locale do you have the CP set to? And does switching it as per the responses linked above fix this?

Thanks,
Andrew

**Comment by HRR1337** [comment]
> Hi @HRR1337 ,
> 
> I've been unable to replicate this using the details provided, doing a little searching around though I've found that other folks have reported that this could be an error with the translation functionality in the control panel ([This issue for example](https://github.com/statamic/cms/issues/1108)).
> 
> What locale do you have the CP set to? And does switching it as per the responses linked above fix this?
> 
> Thanks, Andrew

I'm using multisite:

 'sites' => [

        'default' => [
            'name' => 'English',
            'locale' => 'en_US',
            'url' => '/',
        ],

        'Dutch' => [
            'name' => 'Dutch',
            'locale' => 'nl_NL',
            'url' => '/nl/',
        ],

    ],

removing for example Dutch, wont help.

**Comment by jesperlindstrom** [comment]
We got the same issue. Not multisite. English locale. Changing locale doesn't seem to help.

```
Statamic 3.3.5 Pro
Laravel 8.83.7
PHP 8.1.3
spatie/statamic-responsive-images 2.11.0
withcandour/aardvark-seo 2.0.22
```

**Comment by AndrewHaine** [comment]
Thanks both,

Unfortunately, i've still not managed to replicate this using either environment. It seems that that redirects nav template is causing this so I think we'll just remove this.

It was originally added before Statamic supported adding custom icons to menu items, we'll switch to using the new way of adding icons to menu items which should hopefully fix this.

We're aiming to get a release out of the door by the end of this week to fix this issue and a couple of other 3.3-related issues.

Thanks,
Andrew

**Comment by AndrewHaine** [comment]
Hi both @HRR1337 @jesperlindstrom ,

We've just released v2.0.23 of Aardvark - please update your sites as I believe we've fixed this issue by replacing the custom template that appears to have been causing the issue.

Of course, feel free to re-open this issue if this hasn't fixed the problem!

Thanks,
Andrew

**Comment by jesperlindstrom** [comment]
Seems to work. Thanks for the quick help!
